draft-ietf-quic-recovery.md

```diff
@@ -292,6 +292,10 @@ ack delay field should be ignored.
 Like TCP, QUIC calculates both smoothed RTT and RTT variance similar to those
 specified in {{?RFC6298}}.
 
+A sender might take multiple samples per RTT, and as suggested in {{?RFC6298}},
```

Comment:
```suggestion
Taking a sample from every ACK with a larger largest acked is likely to cause a sender to take multiple samples per RTT, and as suggested in {{?RFC6298}},
```

Reply:
I've rephrased, take a look

Reply:
I really am not in love with the "A sender might" start.